src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public Ball getRandomBall() {
+        ColorSupplier color = new ColorSupplier();
+        String randomColor = color.getRandomColor();
+        Random number = new Random();
+        int randomNumber = number.nextInt(100);
+        Ball randomBall = new Ball(randomColor, randomNumber);
+        return randomBall;

```

Comment:
lets make Random and ColorSupplier global
also dont use magic number - lets make constant for 100 (`private static final`)

Reply:
> lets make Random and ColorSupplier global also dont use magic number - lets make constant for 100 (`private static final`)

How to make Random and ColorSupplier global? Should I just get them out of method and make its static?